README.md

```diff
@@ -1,2 +1,77 @@
-# Module-Two-Capstone-Project
-Using Kanban board with filled-in tasks that have categories and time estimations to assign work in group.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TVMAZE : Movie website:
+![TVMAZE-logo](./src/images/tvm_api.png)
+> This JavaScript capstone project is built on the [TVMAZE API](https://www.tvmaze.com/api), an external API, and Microverse's in-house [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270). The cards display the movies information that was retrieved from the API. The user has the option to like and comment on the sepecific movie.
+
+
+## Built With
+
+- HTML5, JavaScript, and API
+- SASS
+- JEST TEST
+- Linters:
+  - Stylelint, Lighthouse, Webhint and ESLint
+
+## Project Documentation
+
+- Here is the [link]() to our Zoom video presentation
+
+## Website Mockup 📱 💻 🖥️
+![screenshot](./src/images/screen_shoot1.png)
+![screenshot](./src/images/screen_shoot2.png)
+![screenshot](./src/images/screen_shoot3.png)
+
+## Deployment
+- The live version of this web application is deployed here:
+- Link:   https://mukumbuta.github.io/Module-Two-Capstone-Project/
+
+## Live Demo
+https://www.loom.com/share/140ff16e4773437a9d640ab33d98f848
+
```

Comment:
- [ ] It seems that this is your video presentation, but it is recorded via Loom, not via Zoom and has some issues:
-- All the members in the video should appear in the video and talk about the project - Actually, one team member is speaking and also not displaying on the screen. Try to re-record this video via Zoom, with call together and display together on it, as it is required 👍 
 -- The video contains a highlight about a piece of the code the student finds interesting or is proud of making - Please, consider this requirement also because your video is only about the general features of the app, not code sharing and proud piece of it.